cosmoz-default-tree.html

```diff
@@ -146,6 +146,10 @@
 
 	/**
 	 * Returns the nodes on a given path.
+	 * A valid path 1.2.3 should return the items [1, 2, 3]
+	 * - path 1.2.3.3 should return [1, 2, 3, undefined]
+	 * - path 0.1.2.3 should return [1, 2, 3]
+	 * - path 0.1.5.3 should return [1, undefined, undefined]
```

Comment:
It would actually return `[1, undefined, undefined]` because it has lost the path.